docs/static/_redirects

```diff
@@ -24,3 +24,4 @@
 /docs/features/user-management /docs/administration/user-management 301
 /docs/developer/contributing /docs/developer/pr-checklist 301
 /docs/guides/machine-learning /docs/guides/remote-machine-learning 301
+/docs/administration/password-login /docs/administration/system-settings 301
```

Comment:
Reorganizing content is great, but try to remember to add a record here so that we don't end up with broken links due to those changes :pray:

Reply:
Oh, I just read your comment in the description about this. If we have separate PRs for different sections, just include the relevant line immediately. Otherwise, as soon as the PR is merged, the links will be broken.